pynicotine/gtkgui/ui/mainwindow.ui

```diff
@@ -993,7 +993,7 @@
                             <child>
```

Comment:
Suggest `document-edit-symbolic` -> `user-info-symbolic`

![image](https://user-images.githubusercontent.com/88614182/221425816-4fd6742c-de6d-4b2c-b54f-0ebe94eee4bb.png)

![image](https://user-images.githubusercontent.com/88614182/221425846-e547d7d3-c72d-45f9-a9a4-36e026e82b3c.png)

Reply:
Unfortunately this icon is not available in Breeze, and is marked as legacy in the Adwaita icon theme: https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/tree/master/Adwaita/symbolic/legacy

How about `user-home-symbolic`?

Reply:
That works.